### ISSUE TITLE
Log name template character substitution ("LL" compatibility)

### ISSUE THREAD

**Issue by deltecent** [ElvishArtisan/rivendell#611]
The Log Name Template field in RDAdmin's Edit Service dialog allows space characters. The Load Log "LL" RML command does not support spaces in log names.

If a user has a space in their log name template, such as "SHUFFLE FM", the log chain will fail.

Should the Edit Service dialog replace spaces with dashes or underscores ("SHUFFLE_FM") to help users from having log chain failures when creating new services?
```
Aug  6 00:15:51 rdhost ripcd: received rml: "LL 1 SHUFFLE FM-0806 -2!" from 192.168.78.10 (FAILS)
Aug  6 00:15:53 rdhost caed: StopPlayback - Card: 0  Stream: 1  Handle: 61 (DEAD AIR, NOT HAPPY)
Aug  7 00:16:06 rdhost ripcd: received rml: "LL 1 SHUFFLE_FM-0807 -2!" from 192.168.78.10 (WORKS)
Aug  7 00:16:07 rdhost rdairplay: loaded log 'SHUFFLE_FM-0807' in Main Log (NEW LOG)
Aug  7 00:16:07 rdhost caed: LoadPlayback  Card: 0  Stream: 0  Name: /var/snd/060004_008.wav  Handle: 154 (HAPPINESS)
```

**Comment by ElvishArtisan** [comment]
> The Log Name Template field in RDAdmin's Edit Service dialog allows space characters. The Load Log "LL" RML command does not support spaces in log names.
>
> If a user has a space in their log name template, such as "SHUFFLE FM", the log chain will fail.

Correct. This is the intended behavior.


> Should the Edit Service dialog replace spaces with dashes or underscores ("SHUFFLE_FM") to help users from having log chain failures when creating new services?

I would argue that we shouldn't allow whitespace in service names (or any other 'id' field used as a primary key) at all. Whitespace in such identifiers don't play at all well with RML, which uses the ' ' character (ASCII 32) as the field delimiter. This is precisely the reason that we don't allow such whitespace in log names.

**Comment by ElvishArtisan** [comment]
> I would argue that we shouldn't allow whitespace in service names (or any other 'id' field used as a primary key) at all.

Implemented in d433e3c. This does not touch _existing_ service names containing whitespace, but should ensure that no such names are used in future.

**Comment by ElvishArtisan** [comment]
Closing this for now.
